packages/superset-ui-legacy-preset-chart-nvd3/src/NVD3Vis.css

```diff
@@ -153,3 +153,22 @@
   font-size: 14px;
   font-weight: normal;
 }
+
+.d3-tip.nv-event-annotation-layer-table {
+  width: 200px;
+  border-radius: 2px;
+  background-color: #484848;
+  fill-opacity: 0.6;
+  margin: 8px;
+  padding: 8px;
+  color: #fff;
+}
+
+.d3-tip.nv-event-annotation-layer-table::after {
+  content: '\25BC';
+  font-size: 22px;
+  color: #484848;
+  position: absolute;
+  bottom: -23px;
+  left: 84px;
+}
```

Comment:
Here seems to miss a new line. The arrow also looks larger than the old UI. Everything else looks good to me!

Reply:
fixed new line. also make down arrow smaller. thanks!